src/Task/index.js

```diff
@@ -12,20 +12,21 @@ const TaskIcon = ({ variant }) => {
   }
 }
 
-const TaskTitle = ({ title, htmlTitle }) => {
-  return  htmlTitle ? (
-    <div dangerouslySetInnerHTML={{__html: title}} />
-  ) : (
-    <div>{title}</div>
-  )
-}
+const TaskTitle = ({ title, htmlTitle, extra }) => 
+  <div>
+    {htmlTitle ?
+      <span className="ola_task-title-text" dangerouslySetInnerHTML={{__html: title}} /> :
+      <span className="ola_task-title-text">{title}</span>
+    }
+    <div className="ola_task-title-extra">{extra}</div>
```

Comment:
In this new version, with React 16, we should be able to export html fragments, so there's no need to use a `<div>` here, right? @pedroparra @diego-pascual

Reply:
In this case is the container, it's necessary to apply the styles, which I do not know if it is better to put a class or use a selector like is in css, check it. @oscarotero

Reply:
Without that `<div>`, the style is more simple, because the whole task-title is a flex container, so the structure would remain as following:

```html
<div class="ola_task-title">
    <svg class="ola_icon" />
    <div class="ola_task-title-text />
    <div class="ola_task-title-extra />
    <div class="ola_task-icon />
</div>
```

The class `ola_task-title-text` should have the `flex: 1 1 auto`, in order to fill the remaining space.

Doing so, I'd rename `ola_task-title` to `ola_task-header`, so the `ola_task-title-text` could be renamed to `ola_task-title` and `ola_task-title-extra` to simply `ola_task-extra`.